scripts/build.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
```

Comment:
Should we add `-e` to the shebang so we'll fail fast if tsc:build fails?  The build does ultimately fail if there's a ts compilation issue, but, I think that's only because the docs script is failing and happens to be last.

Reply:
Yeah, looking at it now, this `build.sh` file really just turned into a "pre-push" script to ensure all scripts are working. I actually wasn't sure how to make the script fail fast, but `-e` looks like what we need 😄